statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/StatefulFunctionsConfigValidator.java

```diff
@@ -57,10 +57,12 @@ private static void validateParentFirstClassloaderPatterns(Configuration configu
   }
 
   private static Set<String> parentFirstClassloaderPatterns(Configuration configuration) {
-    final String[] split =
-        configuration.get(CoreOptions.ALWAYS_PARENT_FIRST_LOADER_PATTERNS_ADDITIONAL).split(";");
-    final Set<String> parentFirstClassloaderPatterns = new HashSet<>(split.length);
-    for (String s : split) {
+    final String[] patterns =
+        configuration
+            .get(CoreOptions.ALWAYS_PARENT_FIRST_LOADER_PATTERNS_ADDITIONAL)
+            .toArray(new String[0]);
```

Comment:
Probably don't really need `patterns` to be a `String[]`. Fine to just let it be a `List<String>` to keep things simple.

Reply:
Fixed in https://github.com/apache/flink-statefun/pull/319/commits/d2508875511442734ac9296a96354d6f6a0c965a.